### PR TITLE
User/getrou/lottieisland

### PR DESCRIPTION
**Pull request by getrou**
Add LottieIsland to the DrawingIsland Sample

### CODE REVIEW

**Comment by jeffstall** [comment]
I've added a few comments to the PR. Can you update to address them?

File: Samples/Islands/DrawingIsland/DrawingCppTestApp/LottieIslandScenario.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿
+#include "LottieIslandScenario.h"
+
+void CreateLottieIsland()
```
[the remark this answers is not in the document]
Comment (getrou) [reply Samples/Islands/DrawingIsland/DrawingCppTestApp/LottieIslandScenario.cpp]: C++ sample is removed for now, there is an issue with the nuget package. Will try to add it back if I can fix it in time.

File: Samples/Islands/DrawingIsland/DrawingCsTestApp/LottieIslandScenario.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+        if (lottieVisualSource != null)
+        {
+            lottieVisualSource.AnimatedVisualInvalidated += (sender, args) =>
+            {
+                object? diagnostics = null;
+                IAnimatedVisualFrameworkless? animatedVisual = lottieVisualSource.TryCreateAnimatedVisual(compositor, out diagnostics);
+                if (animatedVisual != null && lottieIsland != null)
+                {
+                    compositor.DispatcherQueue.TryEnqueue(() =>
```
[the remark this answers is not in the document]
Comment (jeffstall) [reply Samples/Islands/DrawingIsland/DrawingCsTestApp/LottieIslandScenario.cs]: Oh wow. I wouldn't have expected that AnimatedVisualInvalidated is called from a different thread than the UI, and that TryCreateAnimatedVisual needs to be safe to call from multiple threads.

After build, let's have a task to follow-up on this and fix it. UI objects expect to have events that fire on the UI thread.

File: Samples/nuget.config [hunk 3]
```diff
@@ -9,10 +9,9 @@
     </packageRestore>
     <packageSources>
         <clear />
-        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (jeffstall) [inline Samples/nuget.config:12]: What does removing this do to all of the other NuGets in the repo? For example, how other samples are bringing in WinAppSDK?

**Comment by getrou** [comment]
[heart]         Geoffrey Trousdale (THEY/THEM) reacted to your message:
________________________________

**Pull request merged**